mkbardi@pop-os:~/sem_ws$ ros2 launch shell_simulation shell_simulation.launch.py
[INFO] [launch]: All log files can be found below /home/mkbardi/.ros/log/2025-05-27-16-44-14-338115-pop-os-7826
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [planning_node-1]: process started with pid [7827]
[INFO] [perception_node-2]: process started with pid [7829]
[INFO] [control_node-3]: process started with pid [7831]
[planning_node-1] Traceback (most recent call last):
[planning_node-1]   File "/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/planning_node", line 33, in <module>
[planning_node-1]     sys.exit(load_entry_point('shell-simulation==0.0.0', 'console_scripts', 'planning_node')())
[planning_node-1]   File "/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/planning_node", line 22, in importlib_load_entry_point
[perception_node-2] Traceback (most recent call last):
[perception_node-2]   File "/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/perception_node", line 33, in <module>
[perception_node-2]     sys.exit(load_entry_point('shell-simulation==0.0.0', 'console_scripts', 'perception_node')())
[perception_node-2]   File "/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/perception_node", line 22, in importlib_load_entry_point
[perception_node-2]     for entry_point in distribution(dist_name).entry_points
[perception_node-2]   File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 969, in distribution
[planning_node-1]     for entry_point in distribution(dist_name).entry_points
[planning_node-1]   File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 969, in distribution
[planning_node-1]     return Distribution.from_name(distribution_name)
[planning_node-1]   File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 548, in from_name
[planning_node-1]     raise PackageNotFoundError(name)
[planning_node-1] importlib.metadata.PackageNotFoundError: No package metadata was found for shell-simulation
[perception_node-2]     return Distribution.from_name(distribution_name)
[perception_node-2]   File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 548, in from_name
[perception_node-2]     raise PackageNotFoundError(name)
[perception_node-2] importlib.metadata.PackageNotFoundError: No package metadata was found for shell-simulation
[control_node-3] Traceback (most recent call last):
[control_node-3]   File "/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/control_node", line 33, in <module>
[control_node-3]     sys.exit(load_entry_point('shell-simulation==0.0.0', 'console_scripts', 'control_node')())
[control_node-3]   File "/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/control_node", line 22, in importlib_load_entry_point
[control_node-3]     for entry_point in distribution(dist_name).entry_points
[control_node-3]   File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 969, in distribution
[control_node-3]     return Distribution.from_name(distribution_name)
[control_node-3]   File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 548, in from_name
[control_node-3]     raise PackageNotFoundError(name)
[control_node-3] importlib.metadata.PackageNotFoundError: No package metadata was found for shell-simulation
[ERROR] [planning_node-1]: process has died [pid 7827, exit code 1, cmd '/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/planning_node --ros-args -r __node:=planning_node'].
[ERROR] [perception_node-2]: process has died [pid 7829, exit code 1, cmd '/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/perception_node --ros-args -r __node:=perception_node'].
[ERROR] [control_node-3]: process has died [pid 7831, exit code 1, cmd '/home/mkbardi/sem_ws/install/shell_simulation/lib/shell_simulation/control_node --ros-args -r __node:=control_node'].
mkbardi@pop-os:~/sem_ws$ 